.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: node_js
+node_js:
+ - "5"
```

Comment:
I suppose we can use node 6 and remove before install completely, it will speed up builds a lot